utilities_help/createToolboxMLTBX.m

```diff
@@ -5,7 +5,8 @@ function createToolboxMLTBX(verNumber)
 % project root.
 % Input:
 %   verNumber- string in major.minor.bugfix format
-prjFileName=fullfile("fsdaToolboxPackaging.prj");
```

Comment:
Update the function help text above now that it's a function not a file, you can look at the other MathWorks shipped function's help text to get an idea about the text and formatting.

Reply:
Updated the help text in the MW format.